runtime/middleware/jwt.go

```diff
@@ -0,0 +1,95 @@
+package middleware
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/hypermodeinc/modus/runtime/config"
+	"github.com/hypermodeinc/modus/runtime/logger"
+	"github.com/hypermodeinc/modus/runtime/utils"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JWTClaimsKey string
+
+const JWTClaims JWTClaimsKey = "jwt_claims"
+
+func HandleJWT(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var ctx context.Context = r.Context()
+		tokenStr := r.Header.Get("Authorization")
+		tokenStr = strings.TrimPrefix(tokenStr, "Bearer ")
```

Comment:
This makes the "Bearer " prefix optional.  I believe it should be required.  See https://chatgpt.com/share/6706b29d-85c4-8003-b6a3-abbe79e7a454

Reply:
Ah I’ll update the logic to not be optional but we still need to trim, we can’t verify with the word bearer in front